### PR TITLE
Photorealistic Tiles Demo: Add "orthographic camera" toggle

### PR DESCRIPTION
**Pull request by gkjohnson**
Adds an "orthographic" option to the demo with a transition. I had to make a bug fix in the tiles renderer to support "on demand rendering" with the transition component so that version is updated, as well.

### CODE REVIEW

**Comment by shotamatsuda** [comment]
It works like a charm! So it’s a plugin that makes the transition.

File: storybook/src/atmosphere/PhotorealisticTiles-Story.tsx [hunk 1]
```diff
@@ -128,6 +129,11 @@ const Scene: FC<SceneProps> = ({
 
   const camera = useThree(({ camera }) => camera)
   useLayoutEffect(() => {
+    // Check the camera position to see if we've already moved it to globe surface
+    if (camera.position.length() > 10) {
+      return
+    }
```
Comment (shotamatsuda) [inline storybook/src/atmosphere/PhotorealisticTiles-Story.tsx:135]: What kind of issues did you see without this check? I couldn’t find any on my end.

Comment (gkjohnson) [reply storybook/src/atmosphere/PhotorealisticTiles-Story.tsx:135]: This callback fires every time the camera changes. So when the camera transition animates and changes the camera being rendered with this callback fires again, repositioning the camera and causes a "jump" in the camera position. This check ensures we only position the camera if it's set to it's default position near zero.

**Comment by gkjohnson** [comment]
> It works like a charm! So it’s a plugin that makes the transition.

It's just another R3F component that swaps the camera and transitions it. It wraps the `CameraTransitionManager` class in the 3d tiles project. It's not 3d tiles-specific, though, so it can be used in any project and it's not a plugin in the same sense that `TilesFadePlugin` is a 3d tiles plugin.

File: storybook/src/atmosphere/PhotorealisticTiles-Story.tsx (hunk 1 above)
Comment (shotamatsuda) [reply storybook/src/atmosphere/PhotorealisticTiles-Story.tsx:135]: Sorry, it was just me not noticing the jump in the position!

**Comment by shotamatsuda** [comment]
I moved the transition component and controls into the scene. If you’re comfortable with the change, I’d be happy to merge it!

**Comment by gkjohnson** [comment]
> If you’re comfortable with the change, I’d be happy to merge it!

Looks great to me!

**Comment by shotamatsuda** [comment]
Thanks, I’m merging it. Appreciate your work!

**Pull request merged**